### ISSUE TITLE
Options for Area fill color and opacity

### ISSUE THREAD

**Issue by anhreg** [capitalone/react-native-pathjs-charts#153]
Currently I modify the Line.js at line 103 to make the chart accept area fill color props and opacity

fillOpacity={this.props.options.areaOpacity} stroke="none" fill={ this.props.options.areaColor}. 

I hope the next release we will have it. Thanks a lot!

**Comment by marzolfb** [comment]
Agreed this would be a nice, simple enhancement. Would welcome a PR on it.

**Comment by chrismpalmer** [comment]
Definitely keen for this as well has any progress been made on the pull request?

**Comment by chrismpalmer** [comment]
Friends! I was going to take a stab at this but kind of questions the need for areaColor property in addition to the opacity property. The api allows a specific color and a palette already? Any preferences here?
